test/index.test.js

```diff
@@ -42,7 +42,6 @@ beforeEach(() => {
 
 it('checks audio format support', () => {
   expect(MediaRecorder.isTypeSupported('audio/wav')).toBeTruthy()
-  expect(MediaRecorder.isTypeSupported('audio/wave')).toBeTruthy()
```

Comment:
We still need this test

Reply:
why one is not enough?
unless you check all of them (audio/wave,audio/wav,audio/x-wav,audio/x-pn-wav)
https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types

Reply:
I updated the test btw

Reply:
It will force us to release major update since we broke the previous API.

But OK, it is a good pint that there are many more MIME formats.